source/client/benchmark_client_impl.h

```diff
@@ -46,6 +48,20 @@ struct BenchmarkClientStats {
   ALL_BENCHMARK_CLIENT_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+// For histogram metrics, Nighthawk Statistic is used instead of Envoy Histogram.
+struct BenchmarkClientStatistic {
```

Comment:
What is the relationship between `struct BenchmarkClientStats` and `struct BenchmarkClientStatistic`? They have very similar names, yet don't look directly related.

Can we improve the readability by either or both:
- choosing better name for the new struct, one that would clearly distinguish it from the existing one.
- adding a comment here and above `struct BenchmarkClientStats` explaining the difference.

Reply:
Added a comment explaining the difference.
```
For counter metrics, Nighthawk use Envoy Counter directly. For histogram metrics, Nighthawk use its own Statistic instead of Envoy Histogram. Here BenchmarkClientStats contains only counters while BenchmarkClientStatistic contains only histograms.
```

Reply:
I think the comment helps here, but not elsewhere in the code where this exists.  Could we rename these BenchmarkClientCounters and BenchmarkClientHistograms?  Or something else that distinguishes their purposes in the names themselves.

Reply:
Thanks Nate. Renamed BenchmarkClientStats to BenchmarkClientCounters to avoid confusion (while keep BenchmarkClientStatistic unchanged since some of its members are `StreamingStatistic` which is not exactly histograms)